scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -434,11 +436,16 @@ def synopsis(self):
         if cnt and example_instance:
             logger.info("")
             logger.info("To rerun the tests, call twister using the following commandline:")
-            logger.info("west twister -p <PLATFORM> -s <TEST ID>, for example:")
+            cmd_example = f"west twister -p <PLATFORM> -s <TEST ID>"
+            cmd_explicite = f"west twister -p {example_instance.platform.name} -s {example_instance.testsuite.name}"
+            if not detailed_test_id:
+                cmd_explicite = " ".join([cmd_explicite, "--no-detailed-test-id"])
+                cmd_example = " ".join([cmd_example, "--no-detailed-test-id"])
+            logger.info(f"{cmd_example}, for example:")
             logger.info("")
-            logger.info(f"west twister -p {example_instance.platform.name} -s {example_instance.testsuite.name}")
+            logger.info(f"{cmd_explicite}")
             logger.info(f"or with west:")
-            logger.info(f"west build -p -b {example_instance.platform.name} -T {example_instance.testsuite.name}")
+            logger.info(f"west build -p -b {example_instance.platform.name} {example_instance.testsuite.source_dir_rel} -T {example_instance.testsuite.id}")
             logger.info("-+" * 40)
```

Comment:
```suggestion
        if cnt and example_instance:
            logger.info("")
            logger.info("To rerun the tests, call twister using the following commandline:")
            extra_parameters = '' if detailed_test_id else ' --no-detailed-test-id'
            logger.info(f"west twister -p <PLATFORM> -s <TEST ID>{extra_parameters}, for example:")
            logger.info("")
            logger.info(f"west twister -p {example_instance.platform.name} -s {example_instance.testsuite.name}"
                        f"{extra_parameters}")
            logger.info(f"or with west:")
            logger.info(f"west build -p -b {example_instance.platform.name} "
                        f"{example_instance.testsuite.source_dir_rel} -T {example_instance.testsuite.name}")
            logger.info("-+" * 40)
```
will receive the same log, but with less changes in the code